packages/auth/package.json

```diff
@@ -7,35 +7,38 @@
     "directory": "packages/auth"
   },
   "license": "MIT",
-  "main": "./dist/index.js",
+  "type": "module",
+  "exports": {
+    "types": "./dist/index.d.ts",
+    "import": "./dist/index.mjs",
+    "default": "./dist/index.cjs"
+  },
   "types": "./dist/index.d.ts",
   "files": [
     "dist"
   ],
   "scripts": {
-    "build": "yarn build:js && yarn build:types",
-    "build:js": "babel src -d dist --extensions \".js,.jsx,.ts,.tsx\"",
+    "build": "tsx ./build.ts && yarn build:types",
     "build:pack": "yarn pack -o redwoodjs-auth.tgz",
     "build:types": "tsc --build --verbose",
     "build:watch": "nodemon --watch src --ext \"js,jsx,ts,tsx\" --ignore dist --exec \"yarn build\"",
     "prepublishOnly": "NODE_ENV=production yarn build",
-    "test": "jest src",
-    "test:watch": "yarn test --watch"
+    "test": "vitest run",
+    "test:watch": "vitest watch"
   },
   "dependencies": {
     "@babel/runtime-corejs3": "7.24.1",
     "core-js": "3.36.1",
-    "react": "18.3.0-canary-a870b2d54-20240314"
+    "react": "18.3.0-canary-a870b2d54-20240314",
+    "tsx": "4.7.1"
   },
   "devDependencies": {
-    "@babel/cli": "7.24.1",
     "@babel/core": "^7.22.20",
```

Comment:
If `@babel/cli` is gone `@babel/core` can probably go too

```suggestion
```

Reply:
Done in [`4fbfa47` (#10417)](https://github.com/redwoodjs/redwood/pull/10417/commits/4fbfa4763b6c143f260f4c109ed55991f136f4f8)